src/Camelot.Images/ConcreteImage.cs

```diff
@@ -0,0 +1,14 @@
+using Avalonia.Media.Imaging;
+using Camelot.Services.Abstractions.Models;
+
+
+namespace Camelot.Images;
+
+public class ConcreteImage : ImageModel
+{
+    public Bitmap Bitmap { get; }
```

Comment:
Is there a way to follow one of approaches mentioned here? https://learn.microsoft.com/en-us/dotnet/core/compatibility/core-libraries/6.0/system-drawing-common-windows-only Current solution looks hacky. Alternative approach would be to get rid of `Camelot.Images` and create separate projects for platform specific stuff instead of touching services/abstractions projects that designed for a business logic and not UI stuff

Reply:
This is the trickiest thing on this PR.
I understand the separation of logic and UI.
But here, I think although its "image", it is part of business logic.
At first I thought as a solution to use Skia, or something like that, but it is actually not necessary.
Avalonia provides all that is necessary, just need some Image class in the abstractions layer.

Reply:
I see. My recommendation is:
1) Create `Camelot.UI.AllPlatforms` and corresponding platform specific projects if needed
2) Move this logic there. `Services` was designed for a business logic and not rendering stuff. Feel free to use Avalonia in new projects instead. It's totally ok to have a different logic for rendering on different platforms (surprisingly it didn't happen before).
3) `Camelot.Images` can be removed in this case and all casts won't be necessary anymore

Reply:
a. "Create Camelot.UI.AllPlatforms" - can u send picture when do u want it in solution ?
b. "Move this logic there" - do u mean all the icons services ?

Reply:
I will actually try to build and push this change today in the evening